Parsers/Sysmon-v9.10-Parser.txt

```diff
@@ -1,5 +1,5 @@
 // KQL Sysmon Event Parser
-// Last Updated Date: Dec 17, 2019
+// Last Updated Date: Gen 7, 2020
```

Comment:
I think you meant Jan 7, 2020 here.  Ashwin will confirm on the other changes, but looks good to me.

Reply:
Right Jan 7, sorry I've used Italian month...